fedimint-core/src/db/mod.rs

```diff
@@ -455,6 +457,9 @@ impl Database {
                     }
                 }
             }
+            let delay = (2u64.pow(curr_attempts.min(7) as u32) * 10).min(1000);
+            let delay = rand::thread_rng().gen_range(delay..(2 * delay));
```

Comment:
nit: `delay` doesn't have a semantic type, better alternatives would be:
* Converting to `Duration` as soon as possible (idk if we can generate a range of `Duration`, but worth a try)
* Give a more descriptive name that includes the unit (ms)